plugins/module_utils/network/eos/eos.py

```diff
@@ -365,16 +365,16 @@ def get_diff(
         )
         return diff
 
-    def load_config(self, config, commit=False, replace=False):
+    def load_config(self, config, timer, commit=False, replace=False):
```

Comment:
Needs to be optional
```suggestion
    def load_config(self, config, timer=None, commit=False, replace=False):
```